src/engine-actions/settle-swap.js

```diff
@@ -0,0 +1,13 @@
+const { settleInvoice } = require('../lnd-actions')
+
+/**
+* Settles the invoice for a swap
+*
+* @param {string} swapHash - hash of the swap
+* @returns {Promise}
+*/
+async function settleSwap (swapHash) {
```

Comment:
should add some tests

Reply:
I couldn't think of much to test, but I added one.